src/UartSerial.cpp

```diff
@@ -156,7 +157,7 @@ UartSerial::begin (
     // Leave all output flags unset
 
     // c_cflag - control modes
-    _tio_config.c_cflag |= CS8;  //TODO: Configuration passed by caller
+    _tio_config.c_cflag |= c_cflags;
```

Comment:
Oh, I think that changed in a6aa2ba. So we can either leave that and turn off what we don't want or we can clear it and explicitly set what we do. My only concern with the former is that there might be other things that we are supposed to turn off that we could miss as opposed to turning on absolutely every bit we feel we need. Seems like it would be better to start our config from scratch rather than inherit whatever it's already at and then turn a bunch of flags off and then some others on. Can you think of a reason why we'd want to inherit our config settings from the original before making changes?